rbi/stdlib/openssl.rbi

```diff
@@ -3686,6 +3686,48 @@ end
 class OpenSSL::Digest::DigestError < OpenSSL::OpenSSLError
 end
 
+module OpenSSL::KDF
+  sig do
+    params(
+      pass: String,
+      salt: String,
+      iterations: Integer,
+      length: Integer,
+      hash: T.any(String, OpenSSL::Digest)
+    )
+    .returns(String)
+  end
+  def self.pbkdf2_hmac(pass, salt:, iterations:, length:, hash:); end
+
+  sig do
+    params(
+      pass: String,
+      salt: String,
+      N: Integer,
+      r: Integer,
+      p: Integer,
+      length: Integer
+    )
+    .returns(String)
+  end
+  def self.scrypt(pass, salt:, N:, r:, p:, length:); end
```

Comment:
This will fix the build error.

```suggestion
      n: Integer,
      r: Integer,
      p: Integer,
      length: Integer
    )
    .returns(String)
  end
  def self.scrypt(pass, salt:, n:, r:, p:, length:); end
```

Reply:
Given that this won't work, and that the method is defined in C, what do you think about removing this sig from the rbi?

Reply:
double-splat it is!